### ISSUE TITLE
Lifecycle statistics

### ISSUE THREAD

**Issue by timgl** [PostHog/posthog#1209]
### Is your feature request related to a problem? Please describe

I want to understand which of my DAUs are
- new users
- current users (e.g. there in the last period)
- resurrected users (users not there in the last period but any period before that)
- Lost/Dormant users (active in previous period but not this one

### Describe the solution you'd like

I'm not sure what the best way of showing this is. It might be a separate chart type (like sessions) or it might just be a "shown_as".

### Describe alternatives you've considered

Click on many data points and try to manually reconcile the differences

### Additional context

It looks like this for Amplitude:
![image](https://user-images.githubusercontent.com/1727427/87425442-f2318f80-c5dd-11ea-880b-ebfa27a02a70.png)

**Comment by paolodamico** [comment]
@timgl I would like to prioritize this, I believe it can help us internally to get an overall health view of our user base.

**Comment by timgl** [comment]
@paolodamico It's on the roadmap under "New analytics views" -> this will be the first one we do. However it's blocked by clickhouse. 

I think you've already done the query manually in SQL, if you have could you share it in this issue?

**Comment by EDsCODE** [comment]
Completed #2460
